contracts/interfaces/gov/proposals/ITokenSaleProposal.sol

```diff
@@ -14,12 +14,14 @@ interface ITokenSaleProposal {
     /// @param BABT indicates that the user must own the BABT token
     /// @param TokenLock indicates that the user must lock a specific amount of tokens in the tier
     /// @param NftLock indicates that the user must lock an nft in the tier
+    /// @param MerkleWhitelist indicates that the user must have whitelist Merkle Proofs
     enum ParticipationType {
```

Comment:
We are using this variable on the graph. Once PR is merged, the ABI has to be updated and checked for backwards-compatibility